validator/src/main/java/com/amazon/aoc/validators/AbstractStructuredLogValidator.java

```diff
@@ -35,53 +35,54 @@
 
 import java.time.Instant;
 import java.time.temporal.ChronoUnit;
+import java.util.HashMap;
 import java.util.HashSet;
 import java.util.List;
+import java.util.Map;
 import java.util.Set;
 
 @Log4j2
 public abstract class AbstractStructuredLogValidator implements IValidator {
-  protected CloudWatchService cloudWatchService;
-  protected final ObjectMapper mapper = new ObjectMapper();
-  protected final String validateType;
-  protected String logGroupName;
-
   private static final int MAX_RETRY_COUNT = 6;
   private static final int QUERY_LIMIT = 500;
   private static final int CHECK_INTERVAL_IN_MILLI = 30 * 1000;
   private static final int CHECK_DURATION_IN_SECONDS = 2 * 60;
 
-  public AbstractStructuredLogValidator(String validateType) {
-    this.validateType = validateType;
-  }
+  protected Map<String, JsonSchema> validateJsonSchema = new HashMap<>();
```

Comment:
Might change it to something like schemas , pending schemas or schemasToValidate (because you remove the item when it passes the validation) ? It seems there is method with same name `validateJsonSchema` as well.

Reply:
Changed to `schemasToValidate`